setup.cfg

```diff
@@ -0,0 +1,55 @@
+[metadata]
```

Comment:
Yeah, this is neat. We should do this for Mesa repo as well. Will work on Poetry.

Reply:
I wanted to use poetry too : ) looking forward